--- matchbox-desktop-0.9.1/data/mb-show-desktop.desktop.orig	2004-12-01 12:22:34.000000000 +0100
+++ matchbox-desktop-0.9.1/data/mb-show-desktop.desktop	2006-12-27 03:02:54.847377319 +0100
@@ -1,9 +1,13 @@
 [Desktop Entry]
+Encoding=UTF-8
 Name=Desktop
+Name[pl]=Pulpit
 Comment=Toggle Desktop Visiblity
+Comment[pl]=Przełączenie widoczności pulpitu
 Exec=matchbox-remote -desktop
 Type=Application
+Terminal=false
 Icon=mbdesktop.png
 X-MB-NoWindow=true
-Categories=Action
-
+Categories=Settings;DesktopSettings;
+# vi:encoding=utf-8
